grc/gui/MainWindow.py

```diff
@@ -361,6 +361,10 @@ def update(self):
                 foreground='black' if page.saved else 'red', ro=' (ro)' if page.get_read_only() else '',
                 title=Utils.encode(file_name or NEW_FLOGRAPH_TITLE),
             ))
+            fpath = page.file_path
```

Comment:
Can't we use `page.file_path` directly here? Do we really need the `fpath`variable?

I am not against it, just wondering.

Reply:
in general i'm not a fan of unnecessary intermediate variables, but it looks like in this case it might be desired to not modify page.file_path?

Reply:
@nickoe I don't want to modify `file_path`, as I it is used elsewhere (`file_path == ''` means it is a created flowgraph that was not saved).
Of course you can do this: `page.set_tooltip(page.file_path if page.file_path else '(unsaved)')`
But I think it's too cumbersome.

Reply:
Ah yeah, right. I agree.